modules/stanford_paragraph_postcard/config/install/core.entity_view_display.paragraph.stanford_postcard.default.yml

```diff
@@ -25,15 +26,28 @@ third_party_settings:
       settings:
         pattern:
           field_templates: default
+          variants:
+            image_orientation:
+              constant_value: has-image-left
+              options:
```

Comment:
I recently removed this portion as its not used anywhere. https://github.com/SU-SWS/ui_patterns/commit/b47b86149264b842cdcf8ac90e480a36dce22383